sql_2g/report_queries/erm/costs/erm_inventory_cost.sql

```diff
@@ -0,0 +1,180 @@
+    inst.instance_format_name,
+    CASE WHEN instform.no_instance_formats = 0 THEN
+             NULL
+         ELSE
+             cast((fintrainvl.transaction_amount + coalesce(inv_adj.transactions_inv_adj_total, 0)) / instform.no_instance_formats AS money)
+        END AS "total_by_format", -- this are costs by invoice_line prorated to the number of given formats including adjustments on invoice level
+    inst.instance_subject,
+    CASE WHEN instsub.no_instance_subjects = 0 THEN
+             NULL
+         ELSE
+             cast((fintrainvl.transaction_amount + coalesce(inv_adj.transactions_inv_adj_total, 0)) / instsub.no_instance_subjects AS money)
+        END AS "total_by_subject" -- this are costs by invoice_line prorated to the number of given subjects including adjustments on invoice level
+FROM
+    invoice_lines AS invl
+        LEFT JOIN invoice_detail AS inv ON invl.invoice_id = inv.invoice_id
+        LEFT JOIN folio_reporting.invoice_adjustments_ext AS inv_adj ON inv_adj.invl_id = invl.id
+        LEFT JOIN folio_reporting.finance_transaction_invoices AS fintrainvl ON fintrainvl.invoice_line_id = invl.id
+        LEFT JOIN po_lines AS pol ON invl.po_line_id = pol.id
+        LEFT JOIN instance_detail AS inst ON pol.instance_id = inst.instance_id
+        LEFT JOIN folio_reporting.po_lines_phys_mat_type AS pol_phys_type ON pol.instance_id = pol_phys_type.pol_id
+        LEFT JOIN folio_reporting.po_lines_er_mat_type AS pol_er_type ON pol.instance_id = pol_er_type.pol_id
+        LEFT JOIN folio_reporting.invoice_lines_adjustments AS invl_adjustments ON invl.id = invl_adjustments.invoice_line_id
+        LEFT JOIN instance_subs AS instsub ON instsub.instance_id = inst.instance_id
+        LEFT JOIN instance_formats AS instform ON instform.instance_id = inst.instance_id
+WHERE
+    ((inv.invoice_payment_date >= (SELECT start_date FROM parameters) AND
+		inv.invoice_payment_date < (SELECT end_date FROM parameters))
+		OR 
+		(((SELECT start_date FROM parameters) IS NULL) 
+			OR ((SELECT end_date FROM parameters) IS NULL)))
+	AND 	
+	((invl.invoice_line_status = (SELECT invoice_line_status FROM parameters)) OR 
+		((SELECT invoice_line_status FROM parameters) = ''))
+	AND 	
+	((pol.order_format = (SELECT po_line_order_format FROM parameters)) OR 
+		((SELECT po_line_order_format FROM parameters) = ''))	
+	AND 	
+	((inst.instance_format_name = (SELECT format_name FROM parameters)) OR 
+		((SELECT format_name FROM parameters) = ''))		
+	AND 	
+	((inst.instance_subject = (SELECT instance_subject FROM parameters)) OR 
+		((SELECT instance_subject FROM parameters) = ''))	
+	AND 	
+	((inst.instance_mode_of_issuance_name = (SELECT mode_of_issuance_name FROM parameters)) OR 
+		((SELECT mode_of_issuance_name FROM parameters) = ''))	
+	AND 	
+	((inst.instance_library_name = (SELECT library_name FROM parameters)) OR 
+		((SELECT library_name FROM parameters) = ''))	
+		
```

Comment:
Delete empty line.

Reply:
This query went accidentally into this PR.